src/Uno.UI/UI/Xaml/WindowManagerInterop.wasm.cs

```diff
@@ -167,6 +208,40 @@ private struct WindowManagerSetElementTransformParams
 
 		#endregion
 
+		#region SetPointerEvents
+
+		internal static void SetPointerEvents(IntPtr htmlId, bool enabled)
+		{
+			if (UseJavascriptEval)
+			{
+				var enabledString = enabled ? "true" : "false";
+
+				var command = "Uno.UI.WindowManager.current.setPointerEvents(" + htmlId + ", " + enabledString + ");";
+				var result = WebAssemblyRuntime.InvokeJS(command);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'result'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5247895)